### PR TITLE
Revert behavior changes 

### PR DESCRIPTION
**Pull request by sfc-gh-sfan**
Please answer these questions before submitting your pull requests. Thanks!

1. What GitHub issue is this PR addressing? Make sure that there is an accompanying issue to your PR.

   Fixes #NNNN

2. Fill out the following pre-review checklist:

   - [ ] I am adding a new automated test(s) to verify correctness of my new code
   - [ ] I am adding new logging messages
   - [ ] I am adding a new telemetry message
   - [ ] I am adding new credentials
   - [ ] I am adding a new dependency

3. Please describe how your code solves the related issue.

   Please write a short description of how your code change solves the related issue.

### CODE REVIEW

**Comment by sfc-gh-sfan** [comment]
cc @sfc-gh-kdama @sfc-gh-mabrennan

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,6 @@
 
 ## 1.1.0 (2023-01-24)
 
-### Behavior Changes
-- `SnowparkSourceCodeExtractionException` is raised when Snowpark fails to extract the source code of a function referenced by UDF registration.
--  In `Session.create_dataframe()`, if `data` is a pandas DataFrame and the optional parameter `schema` is specified, `Session.create_dataframe()` now uses `schema` as the resulting DataFrame's schema.
-
 ### New Features:
 - Added `asc`, `asc_nulls_first`, `asc_nulls_last`, `desc`, `desc_nulls_first`, `desc_nulls_last`, `date_part` and `unix_timestamp` in functions.
 - Enabled creating UDFs that reference non-local functions. This feature is not available inside stored procedures.
```
Comment (sfc-gh-stan) [inline CHANGELOG.md:7]: Please remove line 7 as well.

**Review by sfc-gh-stan — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,13 +2,8 @@
 
 ## 1.1.0 (2023-01-24)
 
-### Behavior Changes
-- `SnowparkSourceCodeExtractionException` is raised when Snowpark fails to extract the source code of a function referenced by UDF registration.
--  In `Session.create_dataframe()`, if `data` is a pandas DataFrame and the optional parameter `schema` is specified, `Session.create_dataframe()` now uses `schema` as the resulting DataFrame's schema.
```
Comment (sfc-gh-yixie) [inline CHANGELOG.md:7]: @sfc-gh-stan I remember you said a bug was fixed. Was it also rolled back with this revert?

Comment (sfc-gh-stan) [reply CHANGELOG.md:7]: Yes.

**Review by sfc-gh-yixie — APPROVED** [review]

**Pull request merged**